vertical-pod-autoscaler/RELEASE.md

```diff
@@ -156,8 +156,8 @@ sure nothing we care about will break if we do.
     [groups.yaml](https://github.com/kubernetes/k8s.io/blob/master/groups/sig-autoscaling/groups.yaml)
     under k8s-infra-staging-autoscaling.
 * Permissions to add images to
-    `k8s.gcr.io/images/k8s-staging-autoscaling/images.yaml` are governed by
-    [OWNERS file](https://github.com/kubernetes/k8s.io/blob/master/k8s.gcr.io/images/k8s-staging-autoscaling/OWNERS).
+    `k8s.io/registry.k8s.io/images/k8s-staging-autoscaling/images.yaml` are governed by
```

Comment:
@raywainman, I think the following is the correct link: https://github.com/kubernetes/k8s.io/blob/main/registry.k8s.io/images/k8s-staging-autoscaling/images.yaml

> k8s.io/registry.k8s.io/images/k8s-staging-autoscaling/images.yaml

the above link gives a `404 Page not found` error.